src/api/image/src/gallery.js

```diff
@@ -0,0 +1,33 @@
+const { getPhotos } = require('./photos');
+
+const start = `<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Telescope Background Images</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/png" href="/?t=png&w=200"/>
+  <style>
+    main {
+      display: flex
+      flex-wrap: wrap;
```

Comment:
not sure whether it's gonna work without semi colon?

Reply:
good catch, fixing...